test/test_runset.py

```diff
@@ -16,14 +16,15 @@ def test_check_retcodes(self):
         exe = os.path.join(DATAFILES_PATH, 'bernoulli' + EXTENSION)
         jdata = os.path.join(DATAFILES_PATH, 'bernoulli.data.json')
         sampler_args = SamplerArgs()
+        chain_ids = [1, 2, 3, 4]  # default
```

Comment:
are these always 1, 2, ... ?

Reply:
yes - in this test, we're constructing the default runset as opposed to trying to run the sampler.